calculator/src/Components/Buttons.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './Buttons.css'; 
+
+const Buttons = ({handleInput, handleEqual, handleClear, handleOp, handleZero }) =>  {
+   
+   return (
+        <div className="calculator-container">
+          
+          <div className ="row">
+                <button  onClick={handleClear}className="buttons" value="clear">AC</button> 
+                <button className="buttons">+/-</button>
```

Comment:
These need to have some functionality. Pro tip is never put broken things on a website. It frustrates users. Better to not include than to include something not working.